core/src/main/java/org/springframework/security/authentication/DefaultAuthenticationEventPublisher.java

```diff
@@ -160,6 +160,25 @@ public void setAdditionalExceptionMappings(Properties additionalExceptionMapping
 		}
 	}
 
+	/**
+	 * Overload of {@link #setAdditionalExceptionMappings(Properties)}}, providing the ability
```

Comment:
Thank you for adding a JavaDoc - for consistency, would you please use the description from the other `setAdditionalExceptionMappings` method? I think the `@param` section you have is find, but we should have the same method description so it's clearer that they provide the same guarantees.